public-cloud/azure/ixia-c-dpdk-azure-mellanox/deployment/Python/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM python:3.10
+
+RUN mkdir /snappi
+WORKDIR /snappi
+
+# copy the dependencies file to the working directory
+COPY requirements.txt .
+
+# install dependencies
+RUN apt-get update
+RUN apt-get install jq -y
+RUN pip install -r requirements.txt
```

Comment:
If we update the controller and TE version we may need to change the requirements.txt content as well. For e.g. the latest release https://github.com/open-traffic-generator/ixia-c/releases/tag/v1.8.0-1 uses snappi `1.8.0`